generators/environment/templates/docker/build.devcloud.yml

```diff
@@ -145,3 +154,7 @@ services:
       NPM_CONFIG_CACHE: /root/.cache/npm
       COMPOSER_CACHE_DIR: /root/.cache/composer
       <% if (proxy.exists) { %>PROXY_VIRTUAL_HOST: ${DOCKER_ENV:-local}-<%= host.devcloud %><% } %>
+      # PHPUnit variables.
+      SIMPLETEST_DB: mysql://admin:admin@db/<%= machineName %>_drupal
+      SIMPLETEST_URL: <%= host.local %>
```

Comment:
host.local is not correct here. See the other references to the URL in this file.